src/httpserver.cpp

```diff
@@ -435,6 +430,21 @@ bool InitHTTPServer()
     return true;
 }
 
+bool UpdateHTTPServerLogging(bool enable) {
+#if LIBEVENT_VERSION_NUMBER >= 0x02010100
+    if (enable) {
+        event_enable_debug_logging(EVENT_DBG_ALL);
+    } else {
+        event_enable_debug_logging(EVENT_DBG_NONE);
+    }
+    return true;
+#else
+    // Can't update libevent logging if version < 02010100
+    logCategories &= ~BCLog::LIBEVENT;
```

Comment:
ok, changed so that the caller updates the logCategories bitfield (happens in two places - ` InitHTTPServer()` and the `logging()` rpc).